app/src/main/res/layout-land/topic_practice_subtopic.xml

```diff
@@ -31,20 +32,19 @@
       android:id="@+id/subtopic_check_box"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_marginStart="40dp"
-      android:layout_marginTop="16dp"
       android:button="@drawable/selector_checkbox"
       android:checked="@{isChecked}"
       android:fontFamily="sans-serif"
-      android:gravity="top"
-      android:paddingStart="20dp"
-      android:paddingEnd="20dp"
+      android:gravity="center_vertical"
       android:text="@{viewModel.subtopic.title}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="16sp"
+      android:drawableStart="@android:color/transparent"
+      android:drawablePadding="20dp"
+      android:minHeight="22dp"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done